scalableminds/webknossos
# Fix histogram scale

This PR fixes the histogram scale. Previously we downscaled the data before applying the logarithm. This leads to a linear scale.
With this fix, the logarithm is applied before scaling the data to the histogram height.

During testing, I found 2 errors in the console produced by the new downscaling modal added by the multi-resolution PR. I just fixed those on the fly.

### URL of deployed dev instance (used for testing):
- https://fixhistogramscale.webknossos.xyz

### Steps to test:
- Open a dataset and look at the histogram. Come this with the master. The small values should now be much higher in the histogram. The details in the data should also be increased for small values.
- As I also fiddled with the modal mentioned above please check whether it looks alright and does not throw errors anymore.
- For that: Open a only tracing that did not get downscaled on your local machine and then open the modal using the orange icon above the segmentation layer settings in the tracing view.

### Issues:
- fixes #4925

------
